ui-v2/app/styles/components/icons/index.scss

```diff
@@ -183,3 +207,7 @@
   @extend %with-cross;
   border-radius: 20%;
 }
+%with-no-healthchecks::before {
+  @extend %with-minus;
+  border-radius: 20%;
+}
```

Comment:
This is the extra little minus icon for when there are no healthchecks for a service